Card/card.css

```diff
@@ -0,0 +1,151 @@
+@import url("../../Utility CSS/navbar.css");
+
+.box {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.card-container {
+  width: 35.2rem;
+  height: 50rem;
+  border: 1px solid var(--border-color);
+  margin: 1rem 1rem;
+  color:black;
+  
+}
+
+
+
+.image-container {
+  position: relative;
+}
+
+.img {
+  height: 35rem;
+  width: 35rem;
+}
+
+.badge {
+  font-size: 1.8rem;
+  font-weight: bold;
+  display: inline-block;
+  background-color: var(--secondary-color);
+  width: 9rem;
+  height: 3rem;
+  padding: 4px 8px;
+  position: absolute;
+  top: 0rem;
+  left: 0rem;
+}
+
+.card-title {
+  height: 3rem;
+  width: 36rem;
+  justify-content: space-between;
+  font-weight: bold;
+}
+
+.wishlist-icon {
+  color: rgb(196, 17, 17);
+  padding: 0 3rem;
+  cursor: pointer;
+}
+
+.card-description {
+  font-size: 1.5rem;
+}
+
+.card-price {
+  font-size: 1.9rem;
+  font-weight: bold;
+}
+
+.card-strikeprice {
+  font-size: 1.9rem;
+  font-weight: lighter;
+}
+
+.card-discount {
+  font-size: inherit;
+  color: var(--btn-secondary-color);
+}
+
+.btn-shopping-cart {
+  align-self: center;
+  background-color: var(--primary-color);
+  /* background-color: transparent; */
+  height: 4rem;
+  width: 18rem;
+  border-radius: 1rem;
+  border: none;
+  font-size: 1rem;
+  cursor: pointer;
+  color: rgb(235, 152, 152);
+}
+
+.dismiss-badge {
+  font-size: 1.8rem;
+  font-weight: bold;
+  display: inline-block;
+  background-color: var(--light-color);
+  width: 3rem;
+  height: 3rem;
+  padding: 4px 8px;
+  position: absolute;
+  top: -0rem;
+  left: 32rem;
+  cursor: pointer;
+}
+
+.card-soldout {
+  position: relative;
+  cursor: pointer;
+}
+
+.card-soldout-text {
+  position: absolute;
+  color: white;
+  font-size: 6.9rem;
+  font-weight: bold;
+  z-index: 9999;
+  height: 50rem;
+  width: 35rem;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.text-card {
+  background-color: pink;
+  font-weight:200;
+  margin: 1rem;
+  height: 25rem;
+  width: 60rem;
+}
+
+.horizontal-card-container {
+  width: 70rem;
+  height: 35rem;
+  border: 2px solid var(--border-color);
+  margin: 1rem 1rem;
+}
+
+.btn-small {
+  display: inline-block;
+  height: 5rem;
+  margin: 1rem;
+  width: 15rem;
+}
+
+.shadow {
+  box-shadow: 10px 10px 45px -13px rgba(0, 0, 0, 0.75);
+}
+.code-container {
+  width: 100%;
+}
+
+.code-iframe{
+  border:0; 
+  transform: scale(1);
+   overflow:hidden;
```

Comment:
Code is not indented properly. Use a code formatter.

Reply:
done